### PR TITLE
Update collect-custom-attributes.mdx

### PR DESCRIPTION
**Pull request by jtduffy**
Add the requirement for the Java Agent API jar to be in the class path.

Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

File: src/content/docs/data-apis/custom-data/custom-events/collect-custom-attributes.mdx [hunk 1]
```diff
@@ -96,6 +96,7 @@ To enable and use custom attributes for APM, follow the procedure for your <Inli
     title="Java"
   >
     Custom attribute collection is enabled by default in Java. You can collect custom attributes using XML and the Java agent APIs. These two methods can be used in conjunction with each other.
+    Note that collecting custom attributes requires that the [New Relic Java API jar](/docs/apm/agents/java-agent/api-guides/guide-using-java-agent-api) be in the application's class path.
```
Comment (rhetoric101) [inline src/content/docs/data-apis/custom-data/custom-events/collect-custom-attributes.mdx:99]: @jtduffy I see most usages of this as a single word. Could we change this to "classpath"?

**Review by rhetoric101 — COMMENTED** [review]
@jtduffy thanks for making this update! I just had one question.

File: src/content/docs/data-apis/custom-data/custom-events/collect-custom-attributes.mdx (hunk 1 above)
Comment (jtduffy) [reply src/content/docs/data-apis/custom-data/custom-events/collect-custom-attributes.mdx]: Done

**Review by rhetoric101 — APPROVED** [review]
@jtduffy thanks for adding this helpful information!

**Pull request merged**